sdk/web-pubsub/web-pubsub/test/hubs.spec.ts

```diff
@@ -76,9 +79,10 @@ describe("HubClient", function () {
 
     it("can broadcast using the DAC", async () => {
       const dacClient = new WebPubSubServiceClient(
-        env.WPS_ENDPOINT,
-        new DefaultAzureCredential(),
-        "simplechat"
+        env.WPS_ENDPOINT ?? "",
```

Comment:
You can instead use the `assertEnvironmentVariable` method that is exposed by the recorder.